BootloaderCorePkg/BootloaderCorePkg.dsc

```diff
@@ -296,6 +296,11 @@
 
   gPlatformModuleTokenSpaceGuid.PcdSmbiosTablesBase  | 0x00000000
   gPlatformModuleTokenSpaceGuid.PcdSmbiosStringsPtr  | 0x00000000
+  gPlatformModuleTokenSpaceGuid.PcdP2sb0Ptr  | 0xFEF10EE0
+  gPlatformModuleTokenSpaceGuid.PcdP2sb1Ptr  | 0xFEF10F34
+  gPlatformModuleTokenSpaceGuid.PcdP2sb2Ptr  | 0xFEF10F88
+  gPlatformModuleTokenSpaceGuid.PcdP2sb3Ptr  | 0xFEF10FDC
+  gPlatformModuleTokenSpaceGuid.PcdP2sb4Ptr  | 0xFEF11030
```

Comment:
Where these values come from? Maybe let the platform provide them in some way if the common code could not read them.

Reply:
I've removed this since it will be updated dynamically, thanks.